diff -rup lohit-tamil-2.4.5/Lohit-Tamil.sfd lohit-tamil-2.4.5_mod/Lohit-Tamil.sfd
--- lohit-tamil-2.4.5/Lohit-Tamil.sfd	2011-07-06 14:00:54.919785258 +0530
+++ lohit-tamil-2.4.5_mod/Lohit-Tamil.sfd	2011-07-06 14:00:35.230773619 +0530
@@ -20,7 +20,7 @@ OS2Version: 1
 OS2_WeightWidthSlopeOnly: 0
 OS2_UseTypoMetrics: 1
 CreationTime: 1160992028
-ModificationTime: 1296121888
+ModificationTime: 1309941033
 PfmFamily: 17
 TTFWeight: 400
 TTFWidth: 5
@@ -284,15 +284,14 @@ ShortTable: maxp 16
 EndShort
 LangName: 1033 "" "" "Regular" "ACE: Automatic Control Equipments Lohit Tamil" "" "Version 2.4.5" 
 Encoding: UnicodeBmp
-Compacted: 1
 UnicodeInterp: none
 NameList: Adobe Glyph List
 DisplaySize: -48
 AntiAlias: 1
 FitToEm: 1
-WinInfo: 0 29 11
+WinInfo: 8344 28 11
 AnchorClass2: "Anchor-0"  "'abvm' Above Base Mark in Tamil lookup 0 subtable" 
-BeginChars: 65649 246
+BeginChars: 65649 247
 
 StartChar: .notdef
 Encoding: 65536 -1 0
@@ -1139,7 +1138,7 @@ StartChar: asterisk
 Encoding: 42 42 13
 Width: 666
 GlyphClass: 2
-Flags: WO
+Flags: W
 LayerCount: 2
 Fore
 SplineSet
@@ -25685,5 +25684,37 @@ SplineSet
  173 179 173 179 173 148 c 0,126,127
 EndSplineSet
 EndChar
+
+StartChar: uni20B9
+Encoding: 8377 8377 246
+Width: 624
+Flags: HW
+LayerCount: 2
+Fore
+SplineSet
+358 500 m 0,0,1
+ 359 509 359 509 359 514 c 1,2,-1
+ 59 514 l 1,3,-1
+ 85 577 l 1,4,-1
+ 346 577 l 1,5,6
+ 340 589 340 589 331 599 c 0,7,8
+ 280 654 280 654 165 654 c 2,9,-1
+ 59 654 l 1,10,-1
+ 85 717 l 1,11,-1
+ 583 717 l 1,12,-1
+ 557 654 l 1,13,-1
+ 367 654 l 1,14,15
+ 406 631 406 631 420 577 c 1,16,-1
+ 583 577 l 1,17,-1
+ 557 514 l 1,18,-1
+ 429 514 l 1,19,20
+ 429 358.613 429 358.613 171 340 c 1,21,-1
+ 507 0 l 1,22,-1
+ 407 0 l 1,23,-1
+ 70 337 l 1,24,-1
+ 70 396 l 1,25,26
+ 345.891 394 345.891 394 358 500 c 0,0,1
+EndSplineSet
+EndChar
 EndChars
 EndSplineFont
